.gitlab-ci.yml

```diff
@@ -4,9 +4,11 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-golang-unittests.yml'
   - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-github-status-updates.yml'
+    file: '.gitlab-ci-check-commits.yml'
```

Comment:
In this specific repository we don't need to check for changelog tags, therefore `check-commits-signoffs` should be used instead of `check-commits`

Reply:
Fixed, thanks!